### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by abelsiqueira**
Updating the CHANGELOG for future updates.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,18 @@
...
 ### Changed
 
 ### Removed
+
+## [0.1.0] - 2022-02-14
+### Added
+- First release with existing code.
+
+[Unreleased]: https://github.com/nlesc-recruit/CUDA-wrappers/compare/v0.1.0...HEAD
+[0.1.0]: https://github.com/nlesc-recruit/CUDA-wrappers/releases/tag/v0.1.0
```
Comment (jspaaks) [inline CHANGELOG.md:19]:
```suggestion
[Unreleased]: https://github.com/nlesc-recruit/CUDA-wrappers/compare/0.1.0...HEAD
[0.1.0]: https://github.com/nlesc-recruit/CUDA-wrappers/releases/tag/0.1.0
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,18 @@
...
 ### Added
+- Zenodo badges
 
 ### Changed
 
 ### Removed
+
+## [0.1.0] - 2022-02-14
+### Added
+- First release with existing code.
```
Comment (jspaaks) [inline CHANGELOG.md:16]: Not sure if it is required, but I like having newlines between section headings and content

**Review by jspaaks — CHANGES_REQUESTED** [review]
minor thing about the tags, as well as a note about markdown style

File: CHANGELOG.md (hunk 1 above)
Comment (abelsiqueira) [reply CHANGELOG.md:19]: Nice catch.

**Comment by abelsiqueira** [comment]
Thanks for the review. I have fixed the issues you mentioned.

**Review by jspaaks — APPROVED** [review]
All good!

**Comment by abelsiqueira** [comment]
Thanks for the review

**Pull request merged**